docs/sphinx/sections/api/apidocs/assets.rst

```diff
@@ -7,6 +7,8 @@ An asset is an object in persistent storage, such as a table, file, or persisted
 
 .. autodecorator:: asset
 
+.. autoclass:: MaterializeResult
```

Comment:
sort of agressive to put it up top here, but a lot of the other entries in this list seem more edge case

Reply:
i agree, but i think it's fine